README.md

```diff
@@ -9,18 +9,20 @@ D-in-D image for GitHub Actions Self-Hoster runner based on [summerwind/actions-
 | summerwind/actions-runner-dind | v2.311.0-ubuntu-20.04 |
 | Golang | 1.21.5 |
 | Buildx | 0.17.1 |
-| Node | 20.17.0 |
+| Node | 20.18.0 |
```

Comment:
I think our repo's `.tool-versions` mention node 20.17.0, so it may not run on node 20.18.0, but I'm not sure

Reply:
I have verified the `.tool-versions`, but there is no mention of [node version](https://github.com/Cognyx/webapp/blob/dev/.tool-versions) @francois-codes  even though we can control the node version from the workflow file during node setup step.